kubernetes/kube.libsonnet

```diff
@@ -612,7 +612,16 @@
 
   LimitRange(name, namespace): $._Object('v1', 'LimitRange', name, namespace=namespace),
 
-  PodDisruptionBudget(name, namespace): $._Object('policy/v1beta1', 'PodDisruptionBudget', name, namespace=namespace),
+  PodDisruptionBudget(name, namespace, app=name): $._Object('policy/v1beta1', 'PodDisruptionBudget', name, namespace=namespace) {
+    spec: {
+      maxUnavailable: 1,
```

Comment:
I don't think we want this to be the default. I think we should probably use a percentage like `50%`

Reply:
Yeah.  That's better.  Will fix.